### PR TITLE
[Behat] Fix error when searching a row with more than one columns

### PR DESCRIPTION
**Pull request by loic425**
| Q               | A
| --------------- | ---
| Bug fix?        | yes |
| New feature?    | no |
| BC breaks?      | no |
| Related tickets | |
| License         | MIT |

On Behat, When you search a row in a table with several columns, the "hasRowFields" method of "TableAccessor" return true on the first column which match with fields argument and doesn't check the other fields . This pull-request fix that problem to check all columns of fields argument.

Below an example which fails with current hasRowFields method. This pull request fix that problem.
```
    @ui
    Scenario: Deleting a game play with comments
        Given there is product "Dream Factory"
        And there is game play of "Dream Factory" played by "kevin@example.com" with 2 comments
        And I want to browse game plays
        When I delete game play of "Dream Factory" played by "kevin@example.com"
        Then I should be notified that it has been successfully deleted
```

```
    /**
     * @When I delete game play of :product played by :customer
     */
    public function iDeleteGamePlayWithTitle(ProductInterface $product, Customer $customer)
    {
        $username = $customer->getUser()->getUsername();
        $productName = $product->getName();

        $this->indexPage->deleteResourceOnPage([
            'name' => $username,
            'product' => $productName,
        ]);
    }
```

### CODE REVIEW

**Comment by loic425** [comment]
@pamil I've seen some tests failed, but I'm afraid these are features which seems to be ok but actually not.

**Comment by loic425** [comment]
@pjedrzejewski @pamil 
I added an example to explain what this pull request fix.

**Comment by loic425** [comment]
@pamil @lchrusciel @pjedrzejewski 
I fix all tests except one I put a todo tag. This is an important fix for behat tests !

File: features/shipping/managing_shipping_methods/shipping_method_validation.feature [hunk 1]
```diff
@@ -37,7 +37,7 @@ Feature: Shipping method validation
         Then I should be notified that zone has to be selected
         And shipping method with name "Food and Beverage Tax Rates" should not be added
 
-    @ui
+    @ui @todo
```
Comment (lchrusciel) [inline features/shipping/managing_shipping_methods/shipping_method_validation.feature]: We should remove `ui` tag from here.

Comment (loic425) [inline features/shipping/managing_shipping_methods/shipping_method_validation.feature]: Do you want me to remove this "ui" tag ?

**Comment by loic425** [comment]
@lchrusciel
yep, i'm gonna open an issue for this scenario.

**Comment by loic425** [comment]
@pjedrzejewski 
Now, there are conflicts on this PR, have i to resolve conflicts them ?

**Comment by pjedrzejewski** [comment]
@loic425 Yes, please rebase this PR and is good to merge. :)

**Comment by loic425** [comment]
@pjedrzejewski 
I'm not rebase-friendly.  Which command ?
`git pull upstream master --rebase`
or 
`git rebase upstream/master`

**Comment by lchrusciel** [comment]
`git rebase upstream/master` or on master `git pull upstream master` and `git rebase master` on this branch

**Comment by loic425** [comment]
@lchrusciel 
I think I failed somewhere :( But i fixed conflicts :)

File: src/Sylius/Behat/Service/Accessor/TableAccessor.php [hunk 2]
```diff
@@ -140,16 +140,19 @@ private function hasRowFields(array $columns, array $fields)
                 return false;
             }
 
+            $searchedValue = (string)$searchedValue;
```
Comment (lchrusciel) [inline src/Sylius/Behat/Service/Accessor/TableAccessor.php]: missing space between `)` and `$`

File: features/shipping/managing_shipping_methods/shipping_method_validation.feature (hunk 1 above)
Comment (lchrusciel) [reply features/shipping/managing_shipping_methods/shipping_method_validation.feature]: It would be awesome if you could

**Comment by lchrusciel** [comment]
Thanks @loic425! 👍

**Pull request merged**